### DISCUSSION TITLE
[Bug]: Collections not supported

### DISCUSSION THREAD

**Discussion by simon-tma** [spatie/tabular-assertions#5] [Ideas]
### What happened?

When attempting to match against a collection, a `TypeError` is thrown

### How to reproduce the bug

```php
$users = \collect([
    ['id' => 20, 'name' => 'John Doe', 'email' => 'john@doe.com'],
    ['id' => 1245, 'name' => 'Jane Doe', 'email' => 'jane@doe.com'],
]);

expect($users)->toMatchTable('
    |   id | name        | email           |
    |   20 | John Doe    | john@doe.com    |
    | 1245 | Jane Doe    | jane@doe.com    |
');
```

### Package Version

1.0.0

### PHP Version

8.3.0

### Laravel Version

10.42.0

**Comment by sebastiandedeyne** [comment]
Hi Simon,

This isn't a Laravel package but a generic PHP package. Therefore we don't have any under the hood casting for collections, at the moment you're expected to pass an array.

**Reply by simon-tma** [reply]
Then change support for `Collection` to support for `\Traversable`.
